tests_integration/test_package.py

```diff
@@ -231,9 +231,9 @@ def test_package_with_capital_letters(self, directory_for_test, runner):
             for file in files
         )
 
-    @pytest.mark.integration
     @pytest.mark.integration
     def test_incorrect_input(self, runner):
+        # TODO: refactor snowpark, so pip error is always thrown
```

Comment:
For who is this TODO? Do we want to create jira ticket for it?

Reply:
Ah, it was for me from the morning, changes are already applied. Removed :D